build.py

```diff
@@ -756,32 +764,42 @@ def dockerfile_prepare_container_linux(argmap, backends):
 # example libcurl only needed for GCS?)
 RUN apt-get update && \
     apt-get install -y --no-install-recommends \
-         libb64-0d \
-         libcurl4-openssl-dev \
-         libre2-5 && \
+            libb64-0d \
```

Comment:
Fix indent

Reply:
If we are looking for consistency in the dockerfile, this apt-get does not align with the rest of the apt-get indents